pennylane/ops/functions/evolve.py

```diff
@@ -64,37 +66,112 @@ def evolve(*args, **kwargs):  # pylint: disable=unused-argument
         </html>
 
     Args:
-        op (.ParametrizedHamiltonian): operator to evolve
+        op (.ParametrizedHamiltonian): Hamiltonian to evolve
 
     Returns:
-        ~pennylane.ops.op_math.evolve.ParametrizedEvolution: evolution operator
+        .ParametrizedEvolution: time evolution :math:`U(t_0, t_1)` of the Hamiltonian
+
+
+    The function takes a :class:`.ParametrizedHamiltonian` and solves the time-dependent Schrodinger equation
+
+    .. math:: \frac{\partial}{\partial t} |\psi\rangle = -i H(t) |\psi\rangle
+
+    It returns a :class:`~.ParametrizedEvolution`, :math:`U(t_0, t_1)`, which is the solution to the time-dependent
+    Schrodinger equation for the :class:`~.ParametrizedHamiltonian`, such that
+
+    .. math:: |\psi(t_1)\rangle = U(t_0, t_1) |\psi(t_0)\rangle
 
-    .. seealso:: :class:`.ParametrizedEvolution`
+    .. note::
+        The :class:`~.ParametrizedHamiltonian` must be Hermitian at all times. This is not explicitly
+        checked; ensuring a correctly defined Hamiltonian is the responsibility of the user.
```

Comment:
I'm noticing a lot of repeated statements. I think that in a lot of instances that's a good thing, especially as someone who's quite unfamiliar with pulse programming, but I feel like this is a bit out of place as it should be assumed that a `ParametrizedHamiltonian` is correctly initialized before `qml.evolve` is used on it. I think it should instead be added to the docstring for `ParametrizedHamiltonian`.

Reply:
good point!